### ISSUE TITLE
Blacklist Tracker Request: https://tracker.jiesen.life:8443/announce

### ISSUE THREAD

**Issue by gfxlight55** [ngosang/trackerslist#413]
The following tracker is being detected and blocked by Bitdefender Total Security as suspicious: 

https://tracker.jiesen.life:8443/announce

Please add this tracker to the Blacklist. 

Thank you!

**Comment by ngosang** [comment]
Done. Duplicate of #402
